boards/hail/chip_layout.ld

```diff
@@ -7,3 +8,11 @@ RAM_ORIGIN  = 0x20000000;
 RAM_LENGTH  = 0x00020000;
 
 MPU_MIN_ALIGN = 8K;
+
+MEMORY
```

Comment:
I think if we do this we can kill off all the variables above too. Doesn't look like they're used anywhere else.

Reply:
Gone.